packages/pwa-kit-react-sdk/src/ssr/universal/components/storefront-preview/index.jsx

```diff
@@ -15,21 +15,35 @@ import {useHistory} from 'react-router-dom'
  *
  * @param {boolean} enabled - flag to turn on/off Storefront Preview feature
  * @param  {function(): string | Promise<string>} getToken - A method that returns the access token for the current user
+ * @param  {Array} experimentalUnsafeAdditionalSearchParams - An array of key/value search params to add when context changes
+ * @param  {boolean} experimentalUnsafeReloadServerSide - if true, will reload the page on server side when context changes
  */
-export const StorefrontPreview = ({enabled = true, getToken}) => {
+export const StorefrontPreview = ({
+    enabled = true,
+    getToken,
+    experimentalUnsafeAdditionalSearchParams = [],
+    experimentalUnsafeReloadServerSide = false
+}) => {
     const history = useHistory()
     let isHostTrusted
     useEffect(() => {
         if (enabled && isHostTrusted) {
             window.STOREFRONT_PREVIEW = {
                 ...window.STOREFRONT_PREVIEW,
                 getToken,
-                navigate: (path, action = 'push', ...args) => {
+                experimentalUnsafeNavigate: (path, action = 'push', ...args) => {
```

Comment:
```suggestion
                navigate: (path, action = 'push', ...args) => {
```
`window.STOREFRONT_PREVIEW` is an implementation detail / undocumented interface, and `navigate` is not configurable by / exposed to the customer, so I don't think we need to prefix it.

Reply:
I prefixed it based on @adamraya request in slack. Please confirm which approach we want

Reply:
I think it serves as a good indicator that this is something that we aim to change in the future. But your point is true @wjhsf , but people do crazy things in Javascript land.

Reply:
If y'all want to keep it prefixed, I'm fine with that. Customers aren't supposed to see it / use it anyway, so it doesn't matter what we call it.